__TESTS__/unit/actions/Color.test.ts

```diff
@@ -0,0 +1,64 @@
+import TransformableImage from '../../../src/transformation/TransformableImage';
+import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
+import {Color} from "../../../src/actions/Actions";
+import Colors from '../../../src/actions/color/Colors';
+import {rgb, namedColor} from "../../../src/actions/color/Color";
+
+const CONFIG_INSTANCE = new CloudinaryConfig({
+  cloud: {
+    cloudName: 'demo'
+  }
+});
+
+describe('Tests for Transformation Action -- Color', () => {
+  it('Ensure namespace is correctly populated', () => {
+    expect(Color.rgb).toEqual(rgb);
+    expect(Color.namedColor).toEqual(namedColor);
+  });
+
+  it('Ensures rgb is accepted as an action to TransformableImage', () => {
+    const tImage = new TransformableImage();
+    // Ensures it compiles and doesn't throw
+    expect(
+      tImage.color(rgb(Colors.AZURE2))
```

Comment:
Please add a test that checks that AZURE2 can be used alone:

`import {AZURE2} from '.../Colors`

Reply:
Do you mean this import AZURE2 = Colors.AZURE2; ?